docs/af5-getting-started/modules/ROOT/pages/configure-axon-server.adoc

```diff
@@ -0,0 +1,208 @@
+
+[,console]
+----
+org.axonframework.commandhandling.CommandExecutionException: Exception while handling command
+Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: UNAVAILABLE
+----
+====
+
+== Configuring your application to use the DCB context
+
+Once you've created the DCB-enabled context in Axon Server, you need to configure your application to use it.
+The only configuration change required is to specify the context name in your Axon Server configuration.
+
+Update your `UniversityAxonApplication` class to override the default Axon Server configuration:
+
+[source,java]
+./src/main/java/io/axoniq/demo/university/UniversityAxonApplication.java
+----
+public class UniversityAxonApplication {
+
+    private static final String CONTEXT = "university"; // <1>
+
+    public static ApplicationConfigurer configurer() {
+        var configurer = EventSourcingConfigurer.create();
+        configurer = CreateCourseConfiguration.configure(configurer);
+
+        // Override default Axon Server configuration to use DCB context
+        configurer.componentRegistry(r -> r.registerComponent(AxonServerConfiguration.class, c -> { // <2>
+            var axonServerConfig = new AxonServerConfiguration();
+            axonServerConfig.setContext(CONTEXT); // <3>
+            return axonServerConfig;
+        }));
+
+        return configurer;
+    }
+
+    // ...existing code...
+}
+----
+
+<1> Define the context name that matches what you created in Axon Server UI
+<2> Override the default `AxonServerConfiguration` component registration
+<3> Set the context to use our DCB-enabled context
+
+This configuration ensures that your application connects to the correct context in Axon Server and can take advantage of the Dynamic Consistency Boundary features.
+
+[NOTE]
+====
+This manual configuration approach will be simplified significantly with the upcoming Spring Boot integration for Axon Framework 5.
+In Spring Boot applications, you'll be able to configure the context through simple application properties without needing to manually register components.
+====
+
+== Testing with Axon Server
+
+One of the great benefits of Axon Framework's design (thanks to the new configuration approach and the test fixtures) is that your tests work seamlessly with both the in-memory Event Store and Axon Server.
+When you run your `AxonTestFixture` tests with the Axon Server connector dependency present, the fixture will automatically use your Axon Server configuration.
+This means your tests now run against the same Event Store technology as production.
+
+Your existing test code doesn't need any changes:
+
+[source,java]
+----
+@BeforeEach
+void beforeEach() {
+    var application = new UniversityAxonApplication();
+    fixture = AxonTestFixture.with(application.configurer()); // <1>
+}
+----
+
+<1> The fixture automatically detects and uses Axon Server when the connector is present
```

Comment:
We should put a note here on the disabling of the enhancer, for if they just want to run with in-memory.

Reply:
I've added.